### PR TITLE
Encrypt outgoing messages with STARTTLS by default

### LINKED ISSUE #79: Encrypt outgoing messages with STARTTLS
Currently STARTTLS for outgoing email is not enabled, as seen in headers of an email sent by a Postfix server configured by `ansible-postfix`:

```
Received: from ansible-postfix-server (ansible-postfix-server [IPv6:redacted])
	by mail.redacted with ESMTP
	for <redacted>;
```

Default should be:

```
smtp_tls_security_level = may
```

Read more: http://www.postfix.org/postconf.5.html#smtp_tls_security_level

### PR DESCRIPTION
**Pull request by theel0ja**
Fixes #79.

### CODE REVIEW

File: templates/etc/postfix/main.cf.j2 [hunk 1]
```diff
@@ -25,6 +25,8 @@ smtpd_use_tls=yes
 smtpd_tls_session_cache_database = btree:${data_directory}/smtpd_scache
 smtp_tls_session_cache_database = btree:${data_directory}/smtp_scache
 
+smtp_tls_security_level == {{ postfix_smtp_tls_security_level }}
```
Comment (tersmitten) [inline templates/etc/postfix/main.cf.j2]: `== > =`

Comment (tersmitten) [reply templates/etc/postfix/main.cf.j2]: Maybe change:

```
smtpd_use_tls=yes
```
to:

```
smtpd_use_tls={{ postfix_smtp_tls_security_level != 'none' | ternary('yes', 'no')}}
```

**Comment by tersmitten** [comment]
Thanks for the effort so far

File: templates/etc/postfix/main.cf.j2 (hunk 1 above)
Comment (theel0ja) [reply templates/etc/postfix/main.cf.j2]: Yep.

**Comment by theel0ja** [comment]
Has been fixed now.

**Comment by agimenez** [comment]
Hey, this should be nice to have. Also, recent Postfix versions are deprecating `smtpd_use_tls` in favor of `postfix_smtp_tls_security_level`.

**Comment by tersmitten** [comment]
See #83

**Comment by agimenez** [comment]
Hi @tersmitten , is this ready to be merged? I think it's fine as is.

**Comment by accumulator** [comment]
ship it!

**Pull request merged**